packages/office-ui-fabric-react/src/components/ActivityItem/ActivityItem.styles.ts

```diff
@@ -65,7 +66,20 @@ export const getStyles = memoizeFunction((
       fontSize: COMPACT_ICON_SIZE,
       lineHeight: COMPACT_ICON_SIZE,
       color: theme.palette.themePrimary,
-      marginTop: '1px'
+      marginTop: '1px',
+      selectors: {
+        '.ms-Persona-imageArea': {
+          marginTop: '-2px',
+          border: '2px solid #fff',
```

Comment:
this needs to use a themed color

Reply:
This has been changed to  `theme.palette.white`